scripts/make-env

```diff
@@ -61,6 +65,11 @@ export OPENBALENA_VPN_SERVER_DH=$(b64file "$VPN_DH")
 export OPENBALENA_VPN_SERVICE_API_KEY=$(randstr 32)
 export OPENBALENA_API_VPN_SERVICE_API_KEY=$(randstr 32)
 export OPENBALENA_REGISTRY_SECRET_KEY=$(randstr 32)
+export OPENBALENA_S3_ENDPOINT=$(printf "https://s3.%s" "${DOMAIN}")
```

Comment:
```suggestion
export OPENBALENA_S3_ENDPOINT="https://s3.${DOMAIN}"
```

Reply:
@richbayliss why did you mark this as resolved? Please either commit the suggestion or leave a comment explaining why you think it's better as it is.

Reply:
Agree with @dfunckt here, I'd missed this in my review sweep. Why the need for the `printf`?